docs/sources/enterprise/reporting.md

```diff
@@ -29,7 +29,7 @@ Currently only Organization Admins can create reports.
 1. Enter report information. All fields are required unless otherwise indicated.
    - **Name -** Name of the report as you want it to appear in the Reports list.
    - **Source dashboard -** Select the dashboard to generate the report from.
-   - **Recipients -** Enter the emails of the people or teams that you want to receive the report.
+   - **Recipients -** Enter the emails of the people or teams that you want to receive the report, separated by semicolons.
```

Comment:
```suggestion
   - **Recipients -** Enter the emails of the people or teams that you want to receive the report, separated by commas.
```

Reply:
Semicolons are the **current** functionality, so we need to document it even though we intend to change it.

Reply:
By "intend to change it", do you mean for a given release?

Reply:
Yes, I think [grafana-enterprise #1210](https://app.zenhub.com/workspaces/grafana-enterprise-5f7f23d9e9cc5c0012cbcb07/issues/grafana/grafana-enterprise/1210) will be released in v8.0, but in the meantime we need to document the current functionality, which is that email addresses need to be separated by semicolons. 

An example of why this should be documented: https://raintank-corp.slack.com/archives/C017631SZFW/p1617092786122600